test-framework/junit5/src/main/java/io/quarkus/test/junit/BasicLoggingEnabler.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.test.junit;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import io.quarkus.bootstrap.logging.InitialConfigurator;
+
+/**
+ * A (global) JUnit callback that enables/sets up basic logging if logging has not already been set up.
+ * <p/>
+ * This is useful for getting log output from non-Quarkus tests (if executed separately or before the first Quarkus test),
+ * but also for getting log output from {@code QuarkusTestResourceLifecycleManagers} etc.
+ * <p/>
+ * This callback can be disabled via {@link #CFGKEY_ENABLED} in {@code junit-platform.properties} or via system property.
+ */
+public class BasicLoggingEnabler implements BeforeAllCallback {
+
+    private static final String CFGKEY_ENABLED = "quarkus.test.enable-basic-logging";
```

Comment:
`quarkus.test.[...]` feels logical and is in line with https://github.com/quarkusio/quarkus/blob/65020cd51c2a6abdb16ccebf2f4140467a87136f/test-framework/junit5/src/main/java/io/quarkus/test/junit/util/QuarkusTestProfileAwareClassOrderer.java#L58-L62

But if used via `-D` this will issue that unknown property warning. A bit irritating, maybe.
Any ideas? Change the prefix?

Reply:
Yeah, we should probably change the prefix to avoid the warning

Reply:
Something like `junit.quarkus.enable-basic-logging`? Naming is hard. 😄

Reply:
Sounds fine to me :)

Reply:
Unless someone else comes up with a better idea I'll change it also in the TestClassOrderer (maybe keeping the old ones for a while).